src/rundeck/templates/configmap.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "rundeck.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: "{{ template "rundeck.name" . }}"
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "rundeck.fullname" . }}
+data:
+  framework.properties: |-
+    #    
+    # framework.properties -
+    #
+    
+    # ----------------------------------------------------------------
+    # Rundeck server connection information
+    # ----------------------------------------------------------------
+    
+    framework.server.name = localhost
+    framework.server.hostname = localhost
+    framework.server.port = 4440
+    framework.server.url = http://localhost:4440
+    
+    # ----------------------------------------------------------------
+    # Installation locations
+    # ----------------------------------------------------------------
+    
+    rdeck.base={{ .Values.env.RUNDECK_INSTALL_DIR }}
+    
+    framework.projects.dir={{ .Values.env.SHARED_STORAGE }}/projects
+    framework.etc.dir={{ .Values.env.RUNDECK_INSTALL_DIR }}/etc
+    framework.var.dir={{ .Values.env.RUNDECK_INSTALL_DIR }}/var
+    framework.tmp.dir={{ .Values.env.RUNDECK_INSTALL_DIR }}/var/tmp
+    framework.logs.dir={{ .Values.env.SHARED_STORAGE }}/logs
+    framework.libext.dir={{ .Values.env.RUNDECK_INSTALL_DIR }}/libext
+    
+    # ----------------------------------------------------------------
+    # SSH defaults for node executor and file copier
+    # ----------------------------------------------------------------
+    
+    framework.ssh.keypath = {{ .Values.env.RSA_KEY_DIR }}/id_rsa
+    framework.ssh.user = {{ .Values.env.RD_USER }}
+    
+    # ssh connection timeout after a specified number of milliseconds.
+    # "0" value means wait forever.
+    framework.ssh.timeout = 0
+    
+    
+    # ----------------------------------------------------------------
+    # Auto generated server UUID: 93be48ac-74f4-45f7-a809-bf6139d8a755
+    # ----------------------------------------------------------------
+    rundeck.server.uuid = 93be48ac-74f4-45f7-a809-bf6139d8a755
+    rundeck.tokens.file={{ .Values.env.SHARED_STORAGE }}/token/tokens.properties
+  realm.properties: |-

```

Comment:
These config files should really be migrated into the values file.  See the stable/envoy chart for an example.